src/index.html

```diff
@@ -8,11 +8,17 @@
   <meta name="keywords" content="ギズジョブ, ギズジョブ医療事務, 医療事務求人, 医療事務">
   <meta name="description" content="ギズジョブは完全無料の求職者向け求人サービスです">
   <title>医療事務求人を探すならギズジョブ</title>
+  <!-- Add the slick-theme.css if you want default styling -->
+  <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.css">
+  <!-- Add the slick-theme.css if you want default styling -->
+  <link rel="stylesheet" type="text/css"
+    href="https://cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick-theme.css">
```

Comment:
不要なコメントは削除して、1行にまとめちゃいましょう

```suggestion
  <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.css">
  <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick-theme.css">
```

Reply:
こちら修正しました！
[dd212e9](https://github.com/gizumo-education/front-coding/pull/182/commits/dd212e97f11a38deaf7bd1c83ba45654ff78b631)